specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -7649,10 +7649,6 @@
           "$ref": "#/definitions/VirtualMachineScaleSetStorageProfile",
           "description": "Specifies the storage settings for the virtual machine disks."
         },
-        "additionalCapabilities": {
```

Comment:
This is a breaking change for a stable version, why is this needed

Reply:
This property was there for a feature in private preview and based on discussion between the dev team/PM teams from compute and storage + folks from SDK, there was an agreement that we want to change this to provide better user experience from API stand-point.

Reply:
I understand, but this is a change to a stable version which has already been published, something that is not allowed. Was this approved by ARM

Reply:
Have we already published powershell/CLIs and other tools based on this API-version? My understanding was that it is still not done and our team is still pushing changes for this new api-version.

Reply:
Confirmed offline that this version has not been released in SDKs, this is good to go